internal/kldevents/eventstream.go

```diff
@@ -245,7 +255,13 @@ func (a *eventStream) update(newSpec *StreamInfo) (spec *StreamInfo, err error)
 	}
 	if a.spec.Type == "websocket" && newSpec.WebSocket != nil {
 		a.spec.WebSocket.Topic = newSpec.WebSocket.Topic
+		distributionMode := strings.ToLower(newSpec.WebSocket.DistributionMode)
```

Comment:
Wonder if it's worth pulling this validation into a function, to avoid having the `distributionMode != "broadcast" && distributionMode != "workloaddistribution"` twice (and potentially forgetting to update one of them in the future)